.changeset/plenty-clocks-matter.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog-backend-module-github': patch
```

Comment:
One more comment - there should be a github-org line in here too if it's changed